x-pack/test/security_solution_cypress/cypress/e2e/explore/guided_onboarding/tour.cy.ts

```diff
@@ -68,45 +68,37 @@ describe('Guided onboarding tour', { tags: ['@ess'] }, () => {
     assertTourStepExist(AlertsCasesTourSteps.pointToAlertName);
   });
 
-  describe.skip(
-    'persists tour steps in flyout on flyout toggle',
-    { tags: ['@ess', '@brokenInServerless'] },
-    () => {
-      const stepsInAlertsFlyout = [
-        AlertsCasesTourSteps.reviewAlertDetailsFlyout,
-        AlertsCasesTourSteps.addAlertToCase,
-        AlertsCasesTourSteps.viewCase,
-      ];
+  describe('persists tour steps in flyout on flyout toggle', { tags: ['@ess'] }, () => {
+    const stepsInAlertsFlyout = [
+      AlertsCasesTourSteps.reviewAlertDetailsFlyout,
+      AlertsCasesTourSteps.addAlertToCase,
+      AlertsCasesTourSteps.viewCase,
+    ];
 
-      const stepsInCasesFlyout = [AlertsCasesTourSteps.createCase, AlertsCasesTourSteps.submitCase];
+    const stepsInCasesFlyout = [AlertsCasesTourSteps.createCase, AlertsCasesTourSteps.submitCase];
 
-      stepsInAlertsFlyout.forEach((step) => {
-        it(`step: ${step}, resets to ${step}`, { tags: ['@ess', '@brokenInServerless'] }, () => {
-          startTour();
-          goToStep(step);
-          assertTourStepExist(step);
-          closeAlertFlyout();
-          assertTourStepNotExist(step);
-          expandFirstAlert();
-          assertTourStepExist(step);
-        });
+    stepsInAlertsFlyout.forEach((step) => {
+      it(`step: ${step}, resets to ${step}`, { tags: ['@ess'] }, () => {
```

Comment:
do we need the `tags` in the nested `it` if we already have defined the `tags` in the parent `describe`?

Reply:
not in this case. you can override the parent, but here its obviously not doing that. i will remove it